modules/ossm-deploying-jaeger.adoc

```diff
@@ -80,12 +82,42 @@ spec:
           resources: {}
 ----
 
+
+[id="ossm-deploying-jaeger-production_{context}"]
+== Production Jaeger deployment (fully customized)
+
+The SMCP supports only minimal Elasticsearch parameters.  To fully customize your production environment and access all of the Elasticsearch configuration parameters, use the Jaeger custom resource to configure Jaeger.
+
+Create and configure your Jaeger instance and set  `spec.addons.jaeger.name` to the name of the Jaeger instance, in this example  `jaeger-production-cr`.
+
+.Control Plane with linked Jaeger production CR
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 1000
+    type: Jaeger
+  addons:
+    jaeger:
+      name: jaeger-production-cr #name of Jaeger CR
```

Comment:
Not sure if this commenting in line is a general SM practice but in core docs, I would probably mark this as a callout note:

```
name: jaeger-production <1>
...
<1> The name of the Jaeger CR.
```

Reply:
I'm going to leave as is for now.  I don't want to have a mix.  I'll add it to the checklist of style things to update globally.